Formula/getsb-cli.rb

```diff
@@ -0,0 +1,21 @@
+# Documentation: http://docs.brew.sh/Formula-Cookbook.html
+#                http://www.rubydoc.info/github/Homebrew/brew/master/Formula
+# PLEASE REMOVE ALL GENERATED COMMENTS BEFORE SUBMITTING YOUR PULL REQUEST!
+
+class GetsbCli < Formula
+  desc "Getsb is a command line tool for sending HTTP request."
```

Comment:
Don’t start the description with the name: `"Command-line tool for sending HTTP requests"`. Could it also be a little more specific? There are a lot of formulae that install command-line tools for sending HTTP requests, including `curl` and `wget`.